server.js

```diff
@@ -23,9 +27,142 @@ app.use(express.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  const hello = {
+    Welcome: 'Hello! This be a Book API',
+    Routes: [{
+      "/books_________________________": 'Get books.',
+      "/books?title='title name'______": 'Get books with matching titles.',
+      "/books?author='author name'____": 'Get books from specific author.',
+      "/books/id/'number'_____________": 'Get books with matching id.',
+      "/books/isbn/'number'___________": 'Get books by unique ISBN nr.',
+      "/books/pages/'number'__________": 'Get books with certain amount of pages.',
+      "/books/language/eng____________": 'Get books written in english.',
+      "/books/language/en-US__________": 'Get books written in US-english.',
+      "/books/rating/'number'_________": 'Get books by rating, number = integer.',
+      "/endpoints_____________________": 'Get API endpoints.'
+    }]
+    
+  }
+  res.send(hello)
+})
+
+// List endpoints
+app.get ('/endpoints', (req, res) => {
+  res.send(listEndpoints(app))
+}
+)
+
+// route for books API
+app.get('/books', (req, res) => {
+  const { title, author } = req.query
+ 
+  let booksToSend = books
+
+  if (title){
+    booksToSend = booksToSend.filter(
+      (item) => item.title.toLowerCase().indexOf(title.toLowerCase()) !== -1 )
+    }
+  if (author){
+    booksToSend = booksToSend.filter(
+      (item) => item.authors.toLowerCase().indexOf(author.toLowerCase()) !== -1 )
+    }
+
+  res.json({
+    response: booksToSend,
+    success: true
+  })
+})
+
+// Specific book from id //books/id/"nr"
+app.get('/books/id/:id', (req, res) => {
+  const { id } = req.params
+
+  const bookNR = books.find(item => item.bookID === +id)
+
+  if (!bookNR) {
+    res.status(404).send('No book found!')
+  } else {
+    res.json(bookNR)
```

Comment:
One suggestion is to be more consistent about the responses. On line 70 - 73 you have one way of writing a response, and here you have another way.